web-ui/modules/i18n.mjs

```
@@ -0,0 +1,758 @@
+        'sessions.time.all': '全部时间',
+        'sessions.time.7d': '近 7 天',
+        'sessions.time.30d': '近 30 天',
+        'sessions.time.90d': '近 90 天'
+        ,
+
+        // Usage panel
+        'usage.overview': 'Usage 概览',
+        'usage.range.aria': 'Usage 时间范围',
+        'usage.range.7d': '近 7 天',
+        'usage.range.30d': '近 30 天',
+        'usage.range.all': '全部',
```

Biome (2.4.11)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)